bifrost/admin.py

```diff
@@ -0,0 +1,31 @@
+from wagtail.contrib.modeladmin.options import (
+    ModelAdmin,
+    ModelAdminGroup,
+    modeladmin_register,
+)
+
+from .files import BifrostFile
+
+
+class BifrostFileAdmin(ModelAdmin):
+    model = BifrostFile
+    menu_label = "Files"
+    menu_icon = "fa-file"
+    menu_order = 290
+    add_to_settings_menu = False
+    exclude_from_explorer = False
+
+
+class BifrostAdmin(ModelAdminGroup):
+    menu_label = "Bifrost"
+    menu_icon = "fa-circle-o-notch"
+    menu_order = 110
+    add_to_settings_menu = False
+    exclude_from_explorer = False
+    items = (BifrostFileAdmin,)
+
+
+modeladmin_register(BifrostAdmin)
+
+# SPDX-License-Identifier: (EUPL-1.2)
+# Copyright © 2019-2020 Simon Prast
```

Comment:
Yes, but not of this Project. Please fix SPDX in all new files.

Reply:
Only that file has this invalid licence tag. All other files of bifrost haven't a licence tag at all.
Therefore I will remove this licence tag instead of adjusting it. In future we can add a licence tag to
all files of bifrost.